### ISSUE TITLE
Text editor not opening (no link available to create new file) after upgrade from NC 16.0.1 to Nextcloud 16.0.2

### ISSUE THREAD

**Issue by kieric82** [nextcloud/text#140]
### Steps to reproduce
Hi, first, thanks for the app, this is very nice!

After upgrading from NC 16.0.1 to NC 16.0.2, when you click on the + button to create a new file or folder, the "new Text file" isn't showing up.
I tried to delete the app and reinstall, no success. FYI the old Text editor still works.

### Expected behaviour
Kind obvious this one ;)

### Actual behaviour
Just not showing up and when I click on previous upgrade created file, it opens in the old text editor.

## Server configuration detail

**Operating system:** Linux 4.15.0-54-generic #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 x86_64

**Webserver:** Apache/2.4.29 (Ubuntu) (apache2handler)

**Database:** mysql 10.1.40

**PHP version:** 

7.2.19-0ubuntu0.18.04.1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, sodium, session, standard, apache2handler, mysqlnd, PDO, xml, apcu, apc, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, gmp, iconv, igbinary, imagick, imap, intl, json, ldap, exif, mysqli, pdo_mysql, pdo_pgsql, pdo_sqlite, pgsql, Phar, posix, pspell, readline, recode, redis, shmop, SimpleXML, smbclient, soap, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, wddx, xmlreader, xmlrpc, xmlwriter, xsl, zip, libsmbclient, Zend OPcache

**Nextcloud version:** 16.0.2 - 16.0.2.1

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.2.0
 - activity: 2.9.1
 - admin_audit: 1.6.0
 - admin_notifications: 1.0.2
 - announcementcenter: 3.5.1
 - apporder: 0.7.1
 - bruteforcesettings: 1.3.0
 - calendar: 1.7.0
 - cloud_federation_api: 0.2.0
 - comments: 1.6.0
 - contacts: 3.1.3
 - dav: 1.9.2
 - deck: 0.6.4
 - end_to_end_encryption: 1.1.0
 - event_update_notification: 0.3.4
 - external: 3.3.0
 - federatedfilesharing: 1.6.0
 - federation: 1.6.0
 - files: 1.11.0
 - files_accesscontrol: 1.6.0
 - files_automatedtagging: 1.6.0
 - files_downloadactivity: 1.5.0
 - files_external: 1.7.0
 - files_pdfviewer: 1.5.0
 - files_retention: 1.5.1
 - files_rightclick: 0.13.0
 - files_sharing: 1.8.0
 - files_trashbin: 1.6.0
 - files_versions: 1.9.0
 - files_videoplayer: 1.5.0
 - firstrunwizard: 2.5.0
 - forms: 1.0.3
 - gallery: 18.3.0
 - groupfolders: 4.0.3
 - issuetemplate: 0.5.0
 - logreader: 2.1.0
 - lookup_server_connector: 1.4.0
 - mail: 0.15.1
 - nextcloud_announcements: 1.5.0
 - notes: 3.0.0
 - notifications: 2.4.1
 - oauth2: 1.4.2
 - onlyoffice: 2.4.0
 - password_policy: 1.6.0
 - previewgenerator: 2.1.0
 - privacy: 1.0.0
 - provisioning_api: 1.6.0
 - radio: 0.6.5
 - rainloop: 6.0.2
 - ransomware_detection: 0.5.2
 - ransomware_protection: 1.4.0
 - richdocuments: 3.3.12
 - serverinfo: 1.6.0
 - sharebymail: 1.6.0
 - socialsharing_email: 1.0.5
 - socialsharing_facebook: 1.0.4
 - spreed: 6.0.2
 - support: 1.0.0
 - survey_client: 1.4.0
 - systemtags: 1.6.0
 - tasks: 0.11.0
 - text: 1.0.0
 - theming: 1.7.0
 - timetracker: 0.0.26
 - twofactor_backupcodes: 1.5.0
 - twofactor_totp: 2.1.2
 - updatenotification: 1.6.0
 - video_converter: 0.1.0
 - workflowengine: 1.6.0
Disabled:
 - encryption
 - files_texteditor
 - recommendations
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "REMOVED SENSITIVE VALUE",
        "REMOVED SENSITIVE VALUE"
    ],
    "overwriteprotocol": "https",
    "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "16.0.2.1",
    "overwrite.cli.url": "REMOVED SENSITIVE VALUE",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "memcache.local": "\\OC\\Memcache\\APCu",
    "memcache.distributed": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379
    },
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "installed": true,
    "maintenance": false,
    "data-fingerprint": "7fc625a4d323413eca8097cdbfc6e10d",
    "loglevel": 2,
    "app_install_overwrite": [
        "admin_notifications",
        "apporder",
        "end_to_end_encryption",
        "radio",
        "photosphereviewer",
        "tasks",
        "spreedme",
        "files_reader"
    ],
    "mail_smtpmode": "smtp",
    "mail_smtpsecure": "tls",
    "mail_sendmailmode": "smtp",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "587",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "updater.release.channel": "stable",
    "default_language": "fr",
    "updater.secret": "***REMOVED SENSITIVE VALUE***",
    "theme": ""
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0

**Operating system:** Linux and Windows

## Logs

<details><summary>Web server error log</summary>

```

```
</details>

<details><summary>Nextcloud log</summary>

```

```
</details>

<details><summary>Browser log</summary>

</details>

**Comment by juliushaertl** [comment]
Can you check the browser log for errors?

**Comment by juliushaertl** [comment]
Also does it make a difference if you have enabled or disabled the old text editor?

**Comment by kieric82** [comment]
It doesn't change anything if the old Text editor is either disabled or not. For the browser, not sure how to get the full log but I got a warning regarding the "Viewer" app, not sure if it would help.
Here's the warning:
Viewer app is not installed files.js:31:10
    <anonyme> files.js:31

**Comment by juliushaertl** [comment]
Ok, I could reproduce with the disabled viewer app. Seems our fallback is not registering the file create actions. Thanks for reporting.

**Comment by kieric82** [comment]
Glad this helped, thank you very much for that great app and I hope to be able to use it again soon. :)

**Comment by juliushaertl** [comment]
Another question, did you manually remove the viewer app from your system? It should be shipped with the release. :confused:

**Comment by kieric82** [comment]
yes, sadly. I tought it was messing with my installation. Now I just finished reinstalling it manually, the previously created .md files are opening in the new editor and I can edit them as well. Altough, when I want to create a new file, it does only offer me the legacy text editor. Please let me know if you can help from there, thanks.

**Comment by juliushaertl** [comment]
Thanks, ok that will also be fixed with #141

**Comment by kieric82** [comment]
Perfect! Thank you again for that app!
